### PR TITLE
feat(cron): add disabled flag to cron jobs

### PR DESCRIPTION
**Pull request by origranot**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/nest/blob/master/CONTRIBUTING.md
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: https://github.com/nestjs/schedule/issues/1053

## What is the new behavior?
There is a new ```disabled``` flag under ```lib/decorators/cron.decorator.ts```.
This flag indicates whether the ```orchestrator``` should add the job to the registry or not.

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: lib/decorators/cron.decorator.ts [hunk 1]
```diff
@@ -29,6 +29,11 @@ export interface CronOptions {
    * If you have code that keeps the event loop running and want to stop the node process when that finishes regardless of the state of your cronjob, you can do so making use of this parameter. This is off by default and cron will run as if it needs to control the event loop. For more information take a look at [timers#timers_timeout_unref](https://nodejs.org/api/timers.html#timers_timeout_unref) from the NodeJS docs.
    */
   unrefTimeout?: boolean;
+
+  /**
+   * This flag indicates whether the job will be executed at all (true by default).
```
Comment (micalevisk) [inline lib/decorators/cron.decorator.ts:34]:
```suggestion
   * This flag indicates whether the job will be executed at all.
   * @defaults true
```

Comment (micalevisk) [reply lib/decorators/cron.decorator.ts:34]: actually, the default is `false`, right? :thinking:

Comment (origranot) [reply lib/decorators/cron.decorator.ts]: My bad 😄

**Review by micalevisk — APPROVED** [review]

**Comment by origranot** [comment]
@micalevisk Hey!

I saw the commit you did about the build process that fails because of npm verison.
Can you re-run this build-and-test action?

**Comment by micalevisk** [comment]
I don't have such permission. You could close & open this PR instead

**Comment by origranot** [comment]
> I don't have such permission. You could close & open this PR instead

Looks like it's not trigger anything.
I guess I will just reopen a new PR.

**Review by Tony133 — APPROVED** [review]

**Pull request merged**